### PR TITLE
Update deployment-install-os.md

### PR DESCRIPTION
**Pull request by helmste**
Adding a note to prevent customers from installing Windows Updates using SConfig which will lead to a deployment failure.

### CODE REVIEW

File: azure-stack/hci/deploy/deployment-install-os.md [hunk 1]
```diff
@@ -83,6 +83,9 @@ To use SConfig, sign in to the server running the Azure Stack HCI operating syst
 
 :::image type="content" source="media/deployment-install-os/azure-stack-hci-sconfig-screen.png" alt-text="Screenshot of the Server Configuration tool interface." lightbox="media/deployment-install-os/azure-stack-hci-sconfig-screen.png":::
 
+> [!IMPORTANT]
+> Do not install Windows Updates using SConfig. Updates are installed as part of the deployment process. Installing updates using SConfig will cause the deployment to fail.
+
```
Comment (alkohli) [inline azure-stack/hci/deploy/deployment-install-os.md:88]: Looks good. Just a minor change: Do not install Windows Updates using SConfig. Updates are installed during the deployment. Installing updates using SConfig causes a deployment failure.

**Review by alkohli — COMMENTED** [review]
A minor change. Looks good otherwise. If you can make the change asap, I will sign off today. I need to add another change to this doc and want to make sure that your PR is merged.

**Comment by helmste** [comment]
Update was made.  Thank you for the suggestion, @alkohli!

**Comment by Jak-MS** [comment]
@alkohli 


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by alkohli** [comment]
@helmste Thanks so much. #sign-off Good to merge.

**Pull request merged**